documentation/src/main/asciidoc/topics/attributes/community-attributes.adoc

```diff
@@ -12,6 +12,7 @@
 // Infinispan version
 //
 :ispn_version: 14.0.0.CR1
+:ispn_version_link: 14.0
```

Comment:
good idea but not necessary since we already have an attribute for this

Reply:
OK. I cannot see an exact attribute for 14.0. I need this attribute for a link.

Do you know the location for the existing attribute for 14.0?

Reply:
Why do you need it?
Use: https://github.com/infinispan/infinispan/blob/main/documentation/pom.xml#L21 That has already {infinispan.base.version} included.

Reply:
I'll need it for the link metadata 

link:{javadocroot}/jmxComponents.html#StateTransferManager[`StateTransferManager` ({brandname} {ispn_version_link} API) ] ?

{javadocroot} renders to 14.0?

Reply:
Do you think it's necessary? The api docs version will be always the same as the Infinispan version. I think it makes sense to mention the version when you reference for example a component that has different versioning... 
Consistency: Look for other references to api docs in the additional resources